### DISCUSSION TITLE
Edge Runtime Version

### DISCUSSION THREAD

**Discussion by paul-brenner** [supabase/supabase#21000] [Questions]
How can I find out what version of the edge runtime is when I deploy a function to supabase (using supabase hosted edge functions). Is it possible to see it somewhere online or is the only option to call `Deno.version` in my function? 

Also, is there anyway to know when the supabase team has pushed an update that would result in a new version of the edge runtime being deployed in production?

EDIT - @nyannyacha pushed a fix that addresses the below!
I currently have a function that fails because of a bug in Deno that was fixed in the past few months. If I switch to the supabase CLI beta, my function runs fine because it is working off a version of Deno that includes the fix, but it fails with the release version of the CLI and when deployed to cloud.

But I have no idea how to watch for when the hosted edge runtime is updated so that I can know it is safe to deploy. Also if I try to check the deno version programmatically it would seem that supabase hasn't implemented that ability completely?  If I log `Deno.version` I get something that looks like:
```
  deno: "supabase-edge-runtime-",
  v8: "11.6.189.12",
  typescript: "5.1.6"
```

on a vanilla version of deno that first line would say something like `deno: 1.39.2` I assume that that extra tailing dash in `supabase-edge-runtime-` may mean that someone was planning to eventually have it include some version number?

**Comment by nyannyacha** [comment]
Hi Paul! It's great to see you back on GitHub, and apologize for the inconvenience 😅
That issue is being tracked in [PR-261](https://github.com/supabase/edge-runtime/pull/261) right now 😋

**Reply by paul-brenner** [reply]
WOW! Amazing. This will be very helpful.

I'm going to edit my above question to re-focus it on asking about how to know if the supabase team has pushed a change that will be reflected in the hosted edge function runtime.

**Comment by nyannyacha** [comment]
@paul-brenner 

It appears to be a bug that no version is followed by the dash.
In fact, the release version number of the edge-runtime should be rendered.
https://github.com/supabase/edge-runtime/releases

EDIT: The PR mentioned also includes a fix for this bug. I don't know when the infra will reflect a release with the PR, but the issue with the version not rendering should be fixed.

```json
{
    "deno": "supabase-edge-runtime-1.33.5 (compatible with Deno v1.39.2)",
    "v8": "11.6.189.12",
    "typescript": "5.1.6"
}
```
